.env.sample

```diff
@@ -10,3 +10,5 @@ NEXT_TELEMETRY_DISABLED=1
 HASURA_PORT=8080
 NEXT_PUBLIC_INFURA_ID=781d8466252d47508e177b8637b1c2fd
 NEXT_PUBLIC_GRAPHQL_URL=http://localhost:8080/v1/graphql
+
+BACKEND_HOST=host.docker.internal:4000
```

Comment:
Note that we only want to do this when running the backend outside of docker. Normally we just want to use the Docker version. I'm thinking we get rid of this since it's kinda hacky and make a simpler way to trigger that function in the backend. One good way would just be to check if the DB is empty, and if so call that function to populate it, that way there's no need for people to do it manually

Reply:
I like this idea a lot!
You're right in regards to docker and how the ports will be different.